modules/olm-updating-index-image.adoc

```diff
@@ -36,11 +36,13 @@ endif::[]
 $ opm index add \
     --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \//<1>
     --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
-    --tag <registry>/<namespace>/<existing_index_image>:<tag> <3>
+    --tag <registry>/<namespace>/<existing_index_image>:<tag> \//<3>
+    --pull-tool podman <4>
 ----
 <1> A comma-separated list of additional bundle images to add to the index.
 <2> The existing index that was previously pushed.
 <3> The image tag that you want the updated index image to have.
+<4> A tool that uses `podman` to pull images for `opm index prune`.
```

Comment:
better to change to bellow:
<4> A tool that is used to pull container images for `opm index add`

Reply:
Addressed